README.md

```diff
@@ -1,18 +1,24 @@
 ## TDR Generated GraphQL
 
-This project generates, packages and deploys to the sonatype nexus the case classes generated by the [sbt-graphql](https://github.com/muuki88/sbt-graphql) project.
+This project generates, packages and deploys the case classes generated by the [sbt-graphql](https://github.com/muuki88/sbt-graphql) project to the sonatype nexus.
 
-These classes will be used by other TDR repositories to communicate with the consignment API.
+These classes are be used by other TDR repositories to communicate with the consignment API.
 
 ### Building locally
-* Add a new query file to `src/main/graphql`
+* If you add a new query file to `src/main/graphql`:
 * Add a schema file generated by the [consignment api](https://github.com/nationalarchives/tdr-consignment-api) project. This will be generated in the `target/sbt-graphql` directory and should be copied into `src/main/resources/schema.graphql`
-* Run `sbt package publishLocal`
+* Run `sbt package publishLocal` - This allows you to use your updated version of the API locally to facilitate work on FE (This helps as sometimes Jenkins can be slow to build and catch up etc.)
 
 ### Releasing
-* Commit and push the new query file in `src/main/graphql`
-* Merge to master. This project will be built by the deploy job for the consignment API but it can be also run manually. Go to the TDR Graphql Code Generation
-job in Jenkins, paste the schema from the consignemnt api repo into the build parameters and build.
+* Commit and push the new query file that sits in `src/main/graphql`
+* Merge to master.
```

Comment:
I know this is existing wording, but could you reword this to something like "Open a pull request, and merge to master once reviewed" so that it's clear that we still do PRs for this project rather than merging directly?